hack/dockerized

```diff
@@ -12,7 +12,8 @@ fi
 
 fail_if_cri_bin_missing
 
-KUBEVIRT_BUILDER_IMAGE="quay.io/kubevirt/builder:2306070730-a3b45da40"
+#KUBEVIRT_BUILDER_IMAGE="quay.io/kubevirt/builder:2306070730-a3b45da40"
+KUBEVIRT_BUILDER_IMAGE="quay.io/ehaas/kubevirt-builder:latest"
```

Comment:
This breaks the stable release branches right? Older branches would pull in the `latest` image unless we pinned them in place with each release.

I'd still like to automate updates to this through https://github.com/kubevirt/kubevirt/pull/9870 and a small update to the underlying job. I'll try to find some time this week to move this effort forward.

Reply:
This change is just for testing.
I mentioned it in the PR description:
> When accepted, the last commit needs to be updated with a formal builder image.

I am not clear how we can automate it properly. It does not make sense to push images every-time someone pushed a change. If we can make it per a label added, then maybe it will make sense.

Reply:
> This change is just for testing. I mentioned it in the PR description:
> 
> > When accepted, the last commit needs to be updated with a formal builder image.

My apologies, I missed this. As @0xFelix has said can we drop this now?

> I am not clear how we can automate it properly. It does not make sense to push images every-time someone pushed a change. If we can make it per a label added, then maybe it will make sense.

ACK yeah valid point, it would be useful to have a trigger instead of contentiously building and hosting these images.

Reply:
> My apologies, I missed this. As @0xFelix has said can we drop this now?

If you think this is good without it, I will ask to create a proper image and I update it.

> ACK yeah valid point, it would be useful to have a trigger instead of contentiously building and hosting these images.

Yea, if we manually trigger a job that pushes this, it will work great.
But we need to think how to avoid its abuse, e.g. only allow approvers do it or some other logic.
At the moment, even the manual make target does not work for me locally. I need to do hack it to generate only for one architecture.